storefront/app/controllers/workarea/storefront/content_security_policy.rb

```diff
@@ -0,0 +1,52 @@
+module Workarea
+  module Storefront
+    module ContentSecurityPolicy
```

Comment:
It looks like the Rails DSL supports a block that can be dynamic in a controller. Can we use that instead of the artisanal approach?

Reply:
but the block needs to use the dsl, which would be a nightmare to support via user input.

Reply:
Like because the admin input is just raw text?